docs: fix wording in auto-updater.md (#44787)
electron/electron: docs/api/auto-updater.md

### Event: 'checking-for-update'

> Emitted when checking if an update is available has started.

Reviewer: Not trying to bikeshed too much, but I think this wording is still a little confusing. What do you think of?:
```suggestion
Emitted when checking for an available update has started.
```

I like clarifying the check is for available updates and not for updating itself though 🙂

Author: Works for me! I see what you mean. Thanks for the comment!
